go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -71,6 +71,7 @@ type NodeController struct {
 func NewNode(kube kube.Interface, nodeName string, stopChan <-chan struct{}) (*NodeController, error) {
 	node := &NodeController{
 		kube:       kube,
+		wf:         &factory.WatchFactory{},
```

Comment:
I don't like this; if we need a hack for the unit tests, make the unit tests set a dummy `wf` themselves